recipe/meta.yaml

```diff
@@ -1,21 +1,24 @@
-{% set version = "0.3.0" %}
+{% set version = "1.0.0.dev" %}
 
 package:
   name: geopandas
   version: {{ version }}
 
 source:
-  url: https://github.com/geopandas/geopandas/releases/download/v{{ version }}/geopandas-{{ version }}.tar.gz
-  sha256: e63bb32a3e516d8c9bcd149c22335575defdc5896c8bdf15c836608f152a920b
+  git_url: https://github.com/geopandas/geopandas.git
+  git_branch: geopandas-cython
```

Comment:
@jorisvandenbossche and @mrocklin how about we do this in a simple way then. By using the branch directly all you need to do to create a new package is to bump the build number. The first is up to you, `1.0.0.dev` or `0.4.0.dev`?

Reply:
1.0.0.dev is on the geopandas-cython branch (0.4.0.dev does not exist)